simpleocr/files.py

```diff
@@ -1,9 +1,9 @@
 import os
 import cv2
-from tesseract_utils import read_boxfile, write_boxfile
+from simpleocr.tesseract_utils import read_boxfile, write_boxfile
 
 IMAGE_EXTENSIONS = ['.png', '.tif', '.jpg', '.jpeg']
-DATA_DIRECTORY = 'data/'
+DATA_DIRECTORY = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data")
```

Comment:
It's a bit hacky, but definitely better than the current code in `master` :)
Did you ever take a look at `ResourceManager` from `setuptools`? It seems to be the recommended way to deal with data files. I think I'll try to take a look into it once I have a bit of time - are you in a hurry to get this in?